Examples/Tests/initial_distribution/inputs

```diff
@@ -176,3 +176,9 @@ h4z.bin_number                           = 50
 h4z.bin_min                              = -1.0
 h4z.bin_max                              = +1.0
 h4z.histogram_function(t,x,y,z,ux,uy,uz) = "z"
+
+# our little beam monitor
+bmmntr.type                              = BeamRelevant
```

Comment:
vowels are getting expensive these days :).

Reply:
haha, that's why I needed a comment :D